pkg/config/config.go

```diff
@@ -619,6 +626,16 @@ type Destination struct {
 	IsMachine bool `toml:"is_machine,omitempty"`
 }
 
+// Consumes container image's os and arch and returns list of configured runtimes which
+// can be used for following container image
+func ImagePlatformToRuntimes(os string, arch string, platformToOCI map[string]OCIRuntimeList) []string {
```

Comment:
func (c *EngineConfig)  ImagePlatformToRuntimes(os string, arch string, platformToOCI map[string]OCIRuntimeList) []string {

Reply:
SGTM, I'll fix this in next push.

Reply:
> func (c *EngineConfig) ImagePlatformToRuntimes(os string, arch string, platformToOCI map[string]OCIRuntimeList) 

I agree it should be a method of the config.  That will eliminate the need for the last argument, so we need only two strings: one for arch, one for os.